docs/Installing.asciidoc

```diff
@@ -319,16 +349,42 @@ The openQA web interface can be started via +MOJO_REVERSE_PROXY=1 morbo script/o
 development mode.
 
 +/var/lib/openqa/+ must be owned by root and contain several sub
-directories owned by the user that runs openqa (default 'geekotest'):
+directories, most of which must be owned by the user that runs openQA
+(default 'geekotest'):
 
-* +cache+ (cache directory, needs to be owned by the user who runs openqa)
-* +factory/iso+
-* +share+ (shared directories for remote workers)
-* +testresults+
-* +tools+ (symlink to +/usr/share/openqa/tools/+)
 * +db+ contains the sqlite database
-
-To initially create the database, you need to run +script/initdb+.
+* +images+ is where the server stores test screenshots and thumbnails
+* +share+ contains shared directories for remote workers, can be owned by root
+* +share/factory+ contains test assets and openQA's temporary directory, can be owned by root
+* +share/factory/iso+ contains ISOs for tests
+* +share/factory/hdd+ contains hard disk images for tests
+* +share/factory/repo+ contains repositories for tests
+* +share/factory/other+ contains miscellaneous test assets (e.g. kernels and initrds)
+* +share/factory/tmp+ is used as a temporary directory (openQA will create it if it owns +share/factory+)
+* +share/tests+ contains the tests themselves
+* +testresults+ is where the server stores test logs and test-generated assets
+
+It also contains several symlinks which are necessary due to various things
+moving around over the course of openQA's development. All the symlinks
+can of course be owned by root:
+
+* +script+ (symlink to +/usr/share/openqa/script/+)
+* +tests+ (symlink to +share/tests+)
+* +factory+ (symlink to +share/factory+)
+
+It is always best to use the canonical locations, not the compatibility
+symlinks - so run scripts from +/usr/share/openqa/script+, not
++/var/lib/openqa/script+.
+
+You only need the asset directories for the asset types you will actually use,
+e.g. if none of your tests refer to openQA-stored repositories, you will need
+no +factory/repo+ directory. The distribution packages may not create all
+asset directories, so make sure the ones you need are created if necessary.
+Packages will likewise usually not contain any tests; you must create your
+own tests, or use existing tests for some distribution or other piece of
+software.
+
+To initially create the database, you need to run +/usr/share/openqa/script/initdb+.
```

Comment:
This should no longer be required. Since #574 openQA should be able to auto deploy yourself.